script/generate-git-contribution-report.py

```diff
@@ -0,0 +1,101 @@
+import os
+import requests as github
+import json, yaml
+
+SOURCE_YAML_DATA = '../config/ot-git-details-source.yml'
+OUTPUT_YAML_DATA = 'ot-osc-repo-info.yml'
+
+
+# To Load Input data in yml format
+def _yamlLoader(source_yaml_file):
```

Comment:
instead of this .. use existig lib for yaml loader ..  check this https://github.com/OT-BUILDPIPER-MARKETPLACE/BP-ICEMAN/blob/develop-v1/scripts/schedule_resources.py 
and this https://github.com/OT-BUILDPIPER-MARKETPLACE/BP-ICEMAN/blob/develop-v1/requirements.txt